lib/ansible/parsing/yaml/constructor.py

```diff
@@ -113,7 +139,7 @@ def construct_yaml_seq(self, node):
         data.ansible_pos = self._node_position_info(node)
 
     def construct_yaml_unsafe(self, node):
-        return self.construct_yaml_str(node, unsafe=True)
+        return wrap_var(self.construct_yaml_str(node))
```

Comment:
Nice.  Much better API usage than the previous code :-)

Reply:
previous signatures confused me